pkg/minikube/cruntime/cruntime.go

```diff
@@ -142,6 +144,9 @@ type ListOptions struct {
 	Namespaces []string
 }
 
+// ErrContainerRuntimeNotRunning is thrown when docker runtime is not running
+var ErrContainerRuntimeNotRunning = &oci.FailFastError{Err: errors.New("container runtime is not running")}
```

Comment:
this doesn't need to be a fail fast error type, it could be a normal error type

Reply:
"FailFastError" means an error that cannot try again. Curious why it does not fit here.